configs/sst_i18n-hunspell-dictionaries.yaml

```diff
@@ -121,6 +124,7 @@ data:
   - hunspell-tl
   - hunspell-tn
   - hunspell-tpi
+  - hunspell-tr
```

Comment:
This does not seem to be in c9s (rhel9): so I think you need to fork this file.

Reply:
Ok created -eln suffix workload now